packages/editor-ui/src/components/Node/NodeCreator/viewsData.ts

```diff
@@ -124,7 +126,9 @@ export function AIView(_nodes: SimplifiedNodeType[]): NodeView {
 		value: AI_NODE_CREATOR_VIEW,
 		title: i18n.baseText('nodeCreator.aiPanel.aiNodes'),
 		subtitle: i18n.baseText('nodeCreator.aiPanel.selectAiNode'),
-		info: i18n.baseText('nodeCreator.aiPanel.infoBox'),
+		info: i18n.baseText('nodeCreator.aiPanel.infoBox', {
+			interpolate: { link: templatesStore.getWebsiteCategoryURL('25') },
```

Comment:
would be nice if we had a way to ensure this does not break in the future. since we are breaking apart node categories from workflow categories soon. like referencing by name in the query param. 🤔 thoughts?

Reply:
Good point. Not sure what's the best approach here:
- If we use name than we have to update the website to handle it but again what happens when we change name?
- We could maybe redirect to `https://n8n.io/workflows/?q=ai` which is not ideal but is future-proof

Reply:
maybe we set it to category=ai and as a quick follow up we address this mapping from names..

Reply:
Agree. Since we are already working on the website, we can update it to support category names in urls also.